src/main/java/org/openml/webapplication/fantail/dc/statistical/AttributeEntropy.java

```diff
@@ -69,7 +69,7 @@
 		}
 		
 		Map<String, Double> qualities = new HashMap<String, Double>();
-		if (data.classAttribute().isNominal()) {
+		if (data.classIndex() > 0 && data.classAttribute().isNominal()) {
```

Comment:
Should be >= 0? It is possible that the first feature (id 0) is the target.

Reply:
Nice catch, thanks!